creator-node/src/diskManager.ts

```diff
@@ -561,3 +568,100 @@ export async function sweepSubdirectoriesInFiles(
   // keep calling this function recursively without an await so the original function scope can close
   if (redoJob) return sweepSubdirectoriesInFiles()
 }
+
+/**
+ * Some files have been saved outside the expected `storagePath` location
+ * This job:
+ *  - fetches every file DB record that is not in expected `storagePath`
+ *  - sequentially fetches each file from network and saves to expected `storagePath`, and corrects DB record
+ *  - fetches from network since all files in this scenario are not stored on mounted volume, and so will be cleared on server restart
+ *  - on failure to fetch and store, marks DB record as skipped for later processing
+ */
+export async function fixMisplacedFiles(): Promise<void> {
+  const logger: any = createChildLogger(genericLogger, {
+    function: 'fixMisplacedFiles'
+  })
+
+  // Wrap in a try-catch to retry
+  const startTime = getStartTime()
+  try {
+    const misplacedFileRecords = await models.sequelize.query(
+      `select * from "Files" where "storagePath" not like '${getConfigStoragePath()}%';`
```

Comment:
good question - i think we should do that in a follow up. i think for now we should fix paths relative to env var because then in figments case we would diverge their storage paths into old and new.

I think the best course of action would be to release this with the env var of the storage path, have them migrate to a new node that adheres to the disk format and then force it to be /file_storage/files